### PR TITLE
Run formatter on Logger.ErrorHandler

### PR DESCRIPTION
**Pull request by ruan-brandao**

### CODE REVIEW

File: lib/logger/lib/logger/error_handler.ex [hunk 1]
```diff
@@ -11,9 +11,18 @@ defmodule Logger.ErrorHandler do
     # and then trying to reach it will lead to crashes. So we send a
     # message to a PID, instead of named process, to avoid crashes on
     # send since this handler will be removed soon by the supervisor.
-    {:ok, %{otp: otp?, sasl: sasl?, threshold: threshold,
```
Comment (josevalim) [inline lib/logger/lib/logger/error_handler.ex]: Please move the second argument to a variable.

File: lib/logger/lib/logger/error_handler.ex [hunk 2]
```diff
@@ -48,41 +57,49 @@ defmodule Logger.ErrorHandler do
...
       # Mode is always async to avoid clogging the error_logger
       meta = [pid: ensure_pid(pid), error_logger: ensure_type(type)]
-      :gen_event.notify(state.logger,
-        {level, Process.group_leader(),
-          {Logger, message, Logger.Utils.timestamp(utc_log?), meta}})
+
+      :gen_event.notify(state.logger, {
+        level,
+        Process.group_leader(),
+        {Logger, message, Logger.Utils.timestamp(utc_log?), meta}
```
Comment (josevalim) [inline lib/logger/lib/logger/error_handler.ex]: Please move this to a variable so the call fits on a single line.

File: lib/logger/lib/logger/error_handler.ex [hunk 3]
```diff
@@ -94,22 +111,30 @@ defmodule Logger.ErrorHandler do
   defp ensure_pid(pid) when is_pid(pid), do: pid
   defp ensure_pid(_), do: self()
 
-  defp check_threshold(%{last_time: last_time, last_length: last_length,
-                         dropped: dropped, threshold: threshold} = state) do
-    {m, s, _} = current_time = :os.timestamp
+  defp check_threshold(
+         %{last_time: last_time, last_length: last_length, dropped: dropped, threshold: threshold} =
```
Comment (josevalim) [inline lib/logger/lib/logger/error_handler.ex]: Please remove the pattern matching out of the definition and into the function body.

File: lib/logger/lib/logger/error_handler.ex [hunk 4]
```diff
@@ -94,22 +111,30 @@ defmodule Logger.ErrorHandler do
...
+
       true ->
-        _ = if dropped > 0 do
-          Logger.warn "Logger dropped #{dropped} OTP/SASL messages as it " <>
-                      "exceeded the amount of #{threshold} messages/second"
-        end
+        _ =
+          if dropped > 0 do
+            Logger.warn(
+              "Logger dropped #{dropped} OTP/SASL messages as it " <>
```
Comment (josevalim) [inline lib/logger/lib/logger/error_handler.ex]: Please move error message to a variable.

File: lib/logger/lib/logger/error_handler.ex [hunk 5]
```diff
@@ -48,41 +57,47 @@ defmodule Logger.ErrorHandler do
...
          {:ok, message} <- translate(translators, min_level, level, kind, data, truncate) do
       message = Logger.Utils.truncate(message, truncate)
 
       # Mode is always async to avoid clogging the error_logger
       meta = [pid: ensure_pid(pid), error_logger: ensure_type(type)]
-      :gen_event.notify(state.logger,
-        {level, Process.group_leader(),
-          {Logger, message, Logger.Utils.timestamp(utc_log?), meta}})
+
+      tuple = {Logger, message, Logger.Utils.timestamp(utc_log?), meta}
```
Comment (ruan-brandao) [inline lib/logger/lib/logger/error_handler.ex]: I couldn't think of a better name for this variable. Any suggestions?

**Pull request merged**